automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -38,6 +39,14 @@ object Leonardo extends RestClient with LazyLogging {
   }
 
   object cluster {
+    private case class InvalidCreateClusterApiVersionException(googleProject: GoogleProject,
```

Comment:
I don't see any usages of `InvalidCreateClusterApiVersionException` in this PR? Is it needed? Can it be removed?

Reply:
Ah yea...upon making the return type of `ApiVersion.fromString()` optional, that became unnecessary.